tests/test_validate_call.py

```diff
@@ -12,6 +12,7 @@
 from pydantic import Field, TypeAdapter, ValidationError, validate_call
 
 skip_pre_38 = pytest.mark.skipif(sys.version_info < (3, 8), reason='testing >= 3.8 behaviour only')
+skip_pre_39 = pytest.mark.skipif(sys.version_info < (3, 9), reason='testing >= 3.9 behaviour only')
```

Comment:
We have similar checks in other tests modules as well. I think, we could extract this useful syntax sugar into `conftest.py`.

Reply:
I'm not sure how to do it nicely though:)